### PR TITLE
EDITORIAL CLOSING: Newsletter March 2021

### PR DESCRIPTION
**Pull request by Echolon**
### Final release is planned for **5th of April 2021**.
---
You can make a simple comment or create a merge request to make things easy going for review. You don't need any special technical knowledge to contribute. If you need help, please ask for help in the Communication-Team MUC: commteam@muc.xmpp.org

- [You can use our online Pad](https://yopad.eu/p/xmpp-newsletter-365days)! Please be tidy and write something instead of dropping bare HTTP links :pray: 
- Just paste your prepared text in this thread below or in our XMPP channel (commteam@muc.xmpp.org)
- [Make a pull request by editing the file in this repository](https://github.com/xsf/xmpp.org/blob/newsletter-march2021/content/posts/newsletter/2021-04-05.md)

:arrow_right: Everyone participating, please do not just drop bare links - rather try to make a short and decent formulated description (~5 lines) what is the main outcome of your source. You do not need to write formal - be passionate why this motivates you or what you work on! 

**Note: If you create a pull request, which is highly appreciated, please @mention me once your PR is created!** :pray: 

---

This is our planed timeline for this draft:

XMPP Newsletter Timeline

Status:
- [x] Collection of news
- [x] Review
- [ ] Blog post + Tiny Newsletter
- [ ] Social media
- [ ] Translations
- [ ] Archived ([Use pandoc!](https://pandoc.org))

### CODE REVIEW

File: content/posts/newsletter/2021-04-05.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+JSXC - the Javascript XMPP Client got a [funding for group chat calls!](https://twitter.com/jsxc_org/status/1366352995508748294)
+
+[Monal is now on Mastodon](https://fosstodon.org/@Monal)! The [second beta release](https://monal.im/blog/monal-5-beta-2-out/) has been published as well, high-way to Monal 5.0! Besides, a setup has been created to [support the vaccination process](https://vaccine.monal.im/) in many US states, but also Puerto Rico. More than one million notifications have been sent already. XMPP is once again supporting in the pandemic crisis. There were [several blog posts in March] on this topic, you can start with this selected one](https://monal.im/blog/xmpp-ending-this-pandemic-part-1/). Stay tuned and please support by testing the current beta!
```
Comment (therealjeybe) [inline content/posts/newsletter/2021-04-05.md:62]: > [several blog posts in March]

Is there an link missing?

Comment (Echolon) [reply content/posts/newsletter/2021-04-05.md:62]: Yes, thanks!

Comment (Echolon) [reply content/posts/newsletter/2021-04-05.md]: https://github.com/xsf/xmpp.org/commit/0a3640747bd8d34bcfef1c087223ccb60cb568c8

File: content/posts/newsletter/2021-04-05.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+JSXC - the Javascript XMPP Client got a [funding for group chat calls!](https://twitter.com/jsxc_org/status/1366352995508748294)
+
+[Monal is now on Mastodon](https://fosstodon.org/@Monal)! The [second beta release](https://monal.im/blog/monal-5-beta-2-out/) has been published as well, high-way to Monal 5.0! Besides, a setup has been created to [support the vaccination process](https://vaccine.monal.im/) in many US states, but also Puerto Rico. More than one million notifications have been sent already. XMPP is once again supporting in the pandemic crisis. There were [several blog posts in March](https://monal.im/2021/03/) on this topic, you can start with this selected one](https://monal.im/blog/xmpp-ending-this-pandemic-part-1/). Stay tuned and please support by testing the current beta!
+
+qXMPPconsole - a [browser based XMPP console](https://github.com/voger/qXMPPconsole).
```
Comment (vanitasvitae) [inline content/posts/newsletter/2021-04-05.md:66]: This line does not really say what happened to the client. Was it released? Did it just pop into existence?

**Review by vanitasvitae — CHANGES_REQUESTED** [review]

**Comment by Echolon** [comment]
### :hourglass:  **Editorial closing of the XMPP Newsletter March release** 

### :rocket: Merge in a few minutes! Docker build hopefully finishes by 8 pm :

### :pray: Thanks to anubis, Bastoon, emus, jeybe, jonas-l, Julien Jorge, Holger, peetah, Pierre Maziere, Sam Whited, seveso, vanitasvitae, wurstsalat3000 and Ysabeau for their help in creating it!
### :handshake:   ... ehh.. here we go again, please [let it end](https://timetoherd.com/)... :hand: :mask: :hand: 

### :heart: :fireworks: Next release on 5th of May 2021

**Pull request merged**